subprojects/core/src/integTest/groovy/org/gradle/api/internal/project/taskfactory/TaskPropertyNamingIntegrationTest.groovy

```diff
@@ -102,7 +102,7 @@ class TaskPropertyNamingIntegrationTest extends AbstractIntegrationSpec {
         when:
         run "myTask"
         then:
-        output.contains "Input: inputDirectory [inputA.txt, inputB.txt]"
+        output.contains "Input: inputDirectory [inputs]"
```

Comment:
The file tree for an input directory is created by the visitor itself if necessary. Since the visitor used in the test doesn't do any conversion, there is only the root directory instead of the files in the directory.